modules/nf-core/metamdbg/asm/main.nf

```diff
@@ -0,0 +1,58 @@
+process METAMDBG_ASM {
+    tag "${meta.id}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metamdbg:1.0--hdcf5f25_1':
+        'biocontainers/metamdbg:1.0--hdcf5f25_1' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val(input_type)
+
+    output:
+    tuple val(meta), path("contigs.fasta.gz"), emit: contigs
+    tuple val(meta), path("metaMDBG.log")    , emit: log
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
Is this currently being used? Can we get the output to include the prefix in some way, otherwise every time the module is run the files are all the same name.